src/FactoryTrait.php

```diff
@@ -196,13 +196,11 @@ public function factory($seed, $defaults = [], string $prefix = null): object
      *  - If class (with prefix) exists, do prefix.
      *  - don't prefix otherwise.
      *
-     * Example: normalizeClassName('User', 'Model') == 'Model\User';
-     * Example: normalizeClassName(Test\User::class, 'Model') == 'Test\User'; # or as per "use"
-     * Example: normalizeClassName('.\User', 'Model') == 'Model\User';
-     * Example: normalizeClassName('User', 'Model') == 'Model\User'; # if exists, 'User' otherwise
-     *
-     * # If used without namespace:
-     * Example: normalizeClassName(User::class, 'Model') == 'Model\User'; # if exists, 'User' otherwise
+     * Example: normalizeClassName('User') == 'User'; # default -> no prefix
```

Comment:
`normalizeClassName('User') -> 'User'` seems better to me, rules are above, more comments, more distractions...